health.go

```diff
@@ -267,7 +291,58 @@ func GetOSInfo(ctx context.Context, addr string) OSInfo {
 	return osInfo
 }
 
-// GetSysErrors returns system's RAM and swap information.
+// GetSysServices returns info of sys services that affect minio
+func GetSysServices(ctx context.Context, addr string) SysServices {
+	ss := SysServices{
+		NodeCommon: NodeCommon{Addr: addr},
+		Services:   []SysService{},
+	}
+	srv, e := getSELinuxInfo()
+	if e != nil {
+		ss.Error = e.Error()
+	} else {
+		ss.Services = append(ss.Services, srv)
+	}
+
+	return ss
+}
+
+func getSELinuxInfo() (SysService, error) {
+	ss := SysService{Name: SrvSELinux}
+	configFile := "/etc/selinux/config"
+
+	if _, err := os.Stat(configFile); err != nil {
+		if os.IsNotExist(err) {
+			// selinux is not installed.
+			ss.Status = SrvNotInstalled
+			return ss, nil
+		} else {
+			return ss, err
+		}
+	}
+
+	file, err := os.Open(configFile)
+	if err != nil {
```

Comment:
You could avoid `os.Stat()` and do the check like
```go
if errors.Is(err, os.ErrNotExist) {
    ss.Status = SrvNotInstalled
    return ss, nil
}
```

Reply:
Done